app/models/cms/builtin.rb

```diff
@@ -132,23 +134,23 @@ def to_xml(options = {})
 
   # CMS::Builtin::Page
   class Page < CMS::Builtin
+
+    has_data_tag :builtin_page
+
     def to_xml(options = {})
       xml = options[:builder] || Nokogiri::XML::Builder.new
 
-      xml.builtin_page do |x|
+      xml.__send__(self.class.data_tag) do |x|
         unless new_record?
           xml.id id
           xml.created_at created_at.xmlschema
           xml.updated_at updated_at.xmlschema
         end
-
-        # x.title title
-        x.system_name system_name
```

Comment:
I don't remember what I said when we discussed this, but now checking this, I think maybe it makes sense to keep the `system_name` field. The titles of many built-in pages are the same ("New", "Show", "Index"), but the system name does explain what it is for (`applications/new`, `login/new` etc.).